retrieve-invoice.rst

```diff
@@ -0,0 +1,13 @@
+.. _how-to-retrieve-invoice:
+
+How to retrieve invoices
+========================
+
+To retrieve your invoices, go to the organisation seetings of your project and select *invoices*.
+
+.. image:: /images/organisation-settings.png
+   :alt: 'organisation settings'
+   :width: 300
+
+Here you will see the list of invoices, with the *date* issued, payment *status*, payment *period* and the *amount*
+paid. Click on the *invoice number* to view any other details corresponding to each invoice.
```

Comment:
Avoid using the word "click". Better to use "select", "choose", etc. (Reason: not everyone clicks, e.g. a user with a tablet will tap, etc.)

Reply:
Done